azext_vmware/tests/latest/test_vmware_scenario.py

```diff
@@ -83,48 +83,40 @@ class VmwareScenarioTest(AsyncScenarioTest):
         })
 
         # show should throw ResourceNotFound
-        with self.assertRaisesRegexp(CloudError, 'ResourceNotFound'):
+        with self.assertRaisesRegexp(ApiErrorException, 'ResourceNotFound'):
             self.cmd('vmware privatecloud show -g {rg} -n {privatecloud}')
 
-        count = len(self.cmd('vmware privatecloud list -g {rg}').get_output_in_json()['value'])
+        count = len(self.cmd('vmware privatecloud list -g {rg}').get_output_in_json())
         self.assertEqual(count, 0, 'private cloud count expected to be 0')
 
         # begin creating a private cloud
-        self.cmd('vmware privatecloud create -g {rg} -n {privatecloud} --location {loc} --cluster-size 4 --vpc 192.168.50.0/22')
+        self.cmd('vmware privatecloud create -g {rg} -n {privatecloud} --location {loc} --cluster-size 4 --network-block 192.168.48.0/22')

```

Comment:
`privatecloud create` now takes `--network-block` instead of `--vpc` to match the API rename.